lib/tic_tac_toe_elixir.ex

```diff
@@ -1,11 +1,46 @@
+defmodule Board do
+  def print_board(board_values) do
+    list = Regex.scan(~r/.../, get_board_string(board_values))
```

Comment:
**thought:** This looks like a prime opportunity for some piping! 

https://elixir-lang.org/getting-started/enumerables-and-streams.html#the-pipe-operator